app/modules/transcribe/annotations/graphic.directive.js

```diff
@@ -10,7 +10,6 @@ require('./annotations.module.js')
 function graphicAnnotation($rootScope, AnnotationsFactory) {
     var directive = {
         link: graphicAnnotationLink,
-        replace: true,
```

Comment:
OOI what was this replace key meant to achieve? docs?

Reply:
It's quite obscure tbh. If `true`, it gave the possibility of defining a different html component from that where the directive was originally attached to. https://github.com/angular/angular.js/commit/eec6394a342fb92fba5270eee11c83f1d895e9fb